src/modules/missionHelper/i18n/en_AU.json

```diff
@@ -45,27 +45,27 @@
 	"vehicles": {
 		"captions": {
 			"ambulances": "Ambulance | Ambulances",
-			"battalion_chief_vehicles": "Support",
+			"battalion_chief_vehicles": "Support Vehicle | Support Vehicles",
 			"boats": "Boat | Boats",
 			"brush_truck": "Brush Truck | Brush Trucks",
-			"fire_aviation": "Airborne Firefighting | Airborne Firefightings",
+			"fire_aviation": "Airborne Firefighting Vehicle | Airborne Firefighting Vehicles",
 			"firetrucks": "Fire Engine | Fire Engines",
-			"gw_wasserrettung": "SES Unit",
-			"hazmat_vehicles": "HAZMAT | HAZMATS",
-			"heavy_rescue_vehicles": "Rescue",
-			"helicopter": "Air Ambulance",
+			"gw_wasserrettung": "SES Unit | SES Units",
+			"hazmat_vehicles": "HAZMAT Truck | HAZMATS Trucks",
```

Comment:
Should this not be `HAZMAT` without `s` in plural? So `HAZMAT Trucks`

Reply:
Corrected but can you take a look how there is 2 SES and 2 TOG